app/components/status_box.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if small %>
+  <div class="big-number-with-status">
+    <span class="big-number-status">
+      <% if url %>
+        <a class="govuk-link govuk-link--no-visited-state big-number-link" href="<%= url %>">
+      <% end %>
+        <%= number %> <%= label %>
+        <% if percentage %>
+          - <%= percentage %>%
+        <% end %>
+      <% if url %>
+        </a>
+      <% end %>
```

Comment:
Not one for now but I think this could be made a little easier to follow using a method or two on the component class:

Something along the lines of:

```ruby
if url
  govuk_link_to(box_content, url, no_visited_state: true, class: "big-number-link")
else
  box_content
end
```

Reply:
Very good point, I'll follow up with a PR to move to that setup.